conf/audit.yaml

```diff
@@ -37,7 +37,7 @@
 # Slf4jAuditLogger: Configure you logback.xml to specify location of log files and rotation policy. The following
 # optional parameters are accepted:
 # - log_format   - Format of the audit record sent to SLF4J. Fields can be configured with bash-style parameter
-#                  substitution. Supported parameters are ${CLIENT}, ${USER}, ${BATCH_ID}, ${STATUS}, ${OPERATION},
+#                  substitution. Supported parameters are ${CLIENT_IP}, ${USER}, ${BATCH_ID}, ${STATUS}, ${OPERATION},
```

Comment:
Add CLIENT_PORT and COORDINATOR_IP here...

Reply:
Done